### PR TITLE
fix: Dropdown opening when clicked on ChevronDownIcon in disabled state

### PR DESCRIPTION
**Pull request by archie252000**
Fixes: #1345

### CODE REVIEW

**Review by chaitanyadeorukhkar — DISMISSED** [review]

**Comment by chaitanyadeorukhkar** [comment]
Thanks for the PR @archie252000 🙌

**Review by chaitanyadeorukhkar — APPROVED** [review]

**Comment by archie252000** [comment]
@chaitanyadeorukhkar you're welcome!

**Comment by saurabhdaware** [comment]
Thanks for the PR!! @archie252000 can you check the lint error?

**Comment by chaitanyadeorukhkar** [comment]
@archie252000 while you're fixing the prettier error can you also take the latest merge from master once? Trying to see if we can fix the PR title check failing with #1355

**Comment by archie252000** [comment]
@chaitanyadeorukhkar @saurabhdaware sure, I am on it

File: .github/workflows/blade-pr-title-check.yml
Comment (chaitanyadeorukhkar) [inline .github/workflows/blade-pr-title-check.yml:1]: @archie252000 I think you merged #1355 into this. Instead, can you merge the latest master into this PR?

Comment (archie252000) [reply .github/workflows/blade-pr-title-check.yml:1]: I did in fact merge the master branch, I guess my fork was only behind this particular commit.

Comment (chaitanyadeorukhkar) [reply .github/workflows/blade-pr-title-check.yml:1]: Hmmm. I'm wondering why its showing changes from #1355 in your PR diff as well here

Comment (chaitanyadeorukhkar) [reply .github/workflows/blade-pr-title-check.yml:1]: Can you try doing a `git rebase origin/master`?

Comment (archie252000) [reply .github/workflows/blade-pr-title-check.yml:1]: There was another commit to master branch a couple of minutes ago, I merged with master again. Guess everything is right now.

**Review by chaitanyadeorukhkar — APPROVED** [review]

**Review by saurabhdaware — APPROVED** [review]

**Pull request merged**